Dataset audio - Speech Commands

antimora opened the pull request:
speech_commands from https://huggingface.co/datasets/speech_commands

nathanielsimard commented on burn-dataset/src/audio/speech_commands.rs:
```diff
@@ -0,0 +1,225 @@
+///
+/// The labels are 20 target words, silence and other (22 classes).
+///
+/// Note: label class indeces are not continuous because some classes are grouped together.
```
Typo: indeces => indices

nathanielsimard reviewed (approved):
Apart from the typos, everything seems to be good!

antimora commented:
> Apart from the typos, everything seems to be good!

Thanks. I fixed the typo.

antimora commented:
After your comment about testing, I added to add an example into examples folder. I updated the README as well. You can run this to test:

```shell
[burn-dataset]$ cargo run --example speech_commands --features audio
```
